.github/workflows/danger.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: "2.7.1"
+          ruby-version: "3.0.6"
```

Comment:
do we need to upgrade the node version below

Reply:
Good catch. Updated in https://github.com/nimblehq/nimble-survey-web/pull/84/commits/00f35f6803d8f58b3d244f1e36c99255f28393f9